companion.py

```diff
@@ -4,35 +4,40 @@
 import base64
 import csv
 import requests
-from http.cookiejar import LWPCookieJar	# No longer needed but retained in case plugins use it
+from http.cookiejar import LWPCookieJar  # No longer needed but retained in case plugins use it
 from email.utils import parsedate
 import hashlib
-import json
 import numbers
 import os
-from os.path import dirname, isfile, join
+from os.path import join
 import random
 import sys
 import time
 from traceback import print_exc
 import urllib.parse
 import webbrowser
-import zlib
 
 from config import appname, appversion, config
 from protocol import protocolhandler
 
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:
+    _ = lambda x: x # noqa # to make flake8 stop complaining that the hacked in _ method doesnt exist
 
-holdoff = 60	# be nice
-timeout = 10	# requests timeout
-auth_timeout = 30	# timeout for initial auth
+
+holdoff = 60  # be nice
+timeout = 10  # requests timeout
+auth_timeout = 30  # timeout for initial auth
 
 # Currently the "Elite Dangerous Market Connector (EDCD/Athanasius)" one in
 # Athanasius' Frontier account
-CLIENT_ID   = os.getenv('CLIENT_ID') or 'fb88d428-9110-475f-a3d2-dc151c2b9c7a' 	# Obtain from https://auth.frontierstore.net/client/signup
+# Obtain from https://auth.frontierstore.net/client/signup
+CLIENT_ID   = os.getenv('CLIENT_ID') or 'fb88d428-9110-475f-a3d2-dc151c2b9c7a'
```

Comment:
This does or magic. Maybe replace? Undecided.

Reply:
Nah, this one is fine.  When it's just `<possibly false thing> or <alternative>` I'm OK with it.

Reply:
It would be a different matter if we added a way to specify a CAPI CLIENT_ID on the commandline, then it'd be a three way and really require a proper if statement.  But that's for another PR.